test/es6ToJSONTest.js

```diff
@@ -0,0 +1,243 @@
+const fs = require('fs');
+const {expect} = require('chai');
+const Ajv = require('ajv');
+const setup = require('./setup');
+require('babel-register')({
+  presets: [ 'es2015' ]
+});
+
+setup('./test/fixtures/spec', './build/test', true);
+const ajv = setupAjv('./build/test/schema');
+
+describe('#ToJSON', () => {
+  
+  describe('#StringValueEntryClass()', () => {
+    const StringValueEntry = importResult('shr/simple/StringValueEntry');
+    it('should serialize a JSON instance', () => {
+      const json = getJSON('StringValueEntry');
+      const entry = StringValueEntry.fromJSON(json);
+      expect(entry).instanceOf(StringValueEntry);
+
+      let gen_json = entry.toJSON();
+      validateJSON('StringValueEntry', gen_json);
```

Comment:
I didn't notice this before, but... all of these tests only validate that the JSON conforms to the schema.  It seems we should also be checking that the actual values in the JSON are what we expect them to be (e.g., if the instance's `value` field was `'foo'`, then the exported JSON's `'Value'` should also be `'foo'`).  Otherwise we could still have some significant export errors not caught at all.

I'm not sure, but you _might_ be able to simply do a deep equal on the JSON you started with and the JSON you generated:
```
expect(gen_json).to.eql(json);
```
I think the JSON should be round-trippable.